correlation_context/HTTP_HEADER_FORMAT.md

```diff
@@ -1,43 +1,71 @@
 # Correlation Context HTTP Header Format
 
-A correlation context header is used to pass the name-value context properties for the trace. This is a companion header for the `traceparent`. The values should be passed along to any child requests. Note that uniqueness of the key within the `Correlation-Context` is not guaranteed. Context received from upstream service may be altered before passing it along.
+The correlation context header is used to propagate user-supplied key-value pairs through a trace.
+A received header MAY be altered to change or add information and it MUST be passed on to all downstream request.
+
+Multiple correlation context headers are allowed. Values can be combined in a single header according to  [RFC 7230](https://tools.ietf.org/html/rfc7230#page-24).
 
 *See [rationale document](HTTP_HEADER_FORMAT_RATIONALE.md) for details of decisions made for this format.*
 
-# Format
 
-## Header name
+# Header Name
+
+Header name: `Correlation-Context`
+
+@TODO: Agree on the header name
+@TODO: Add more information on casing once we agree on the final header name
 
-`Correlation-Context`
 
-Multiple correlation context headers are allowed. Values can be combined in a single header according to the [RFC 7230](https://tools.ietf.org/html/rfc7230#page-24).
+# Header Content
 
-## Header value
+This section uses the Augmented Backus-Naur Form (ABNF) notation of [[!RFC5234]], including the DIGIT rule in <a data-cite='!RFC5234#appendix-B.1'>appendix B.1 for RFC5234</a>. It also includes the `OWS` rule from <a data-cite='!RFC7230#whitespace'>RFC7230 section 3.2.3</a>.
```

Comment:
```suggestion
This section uses the Augmented Backus-Naur Form (ABNF) notation of [[!RFC5234]], including the DIGIT rule in <a data-cite='!RFC5234#appendix-B.1'>appendix B.1 for RFC5234</a>. It also includes the `OWS` rule (optional whitespace) from <a data-cite='!RFC7230#whitespace'>RFC7230 section 3.2.3</a>.
```

this simple clarification makes it unnecessary to look up the other RFC (whereas DIGIT is self-explanatory)